bindings/node/.evergreen/prebuild.sh

```diff
@@ -5,6 +5,19 @@ if [ -z ${DISTRO_ID+omitted} ]; then echo "DISTRO_ID is unset" && exit 1; fi
 set -o errexit
 set +o xtrace
 
+echo "Setting up environment"
+
+export PATH="/opt/mongodbtoolchain/v2/bin:$PATH"
+hash -r
+
+NODE_LTS_VERSION=${NODE_LTS_VERSION:-16}
+export NODE_LTS_VERSION=${NODE_LTS_VERSION}
+source ./.evergreen/install-dependencies.sh
+
+# install node dependencies
+echo "Installing package dependencies (includes a static build)"
+bash ./etc/build-static.sh
+
```

Comment:
since we explicitly call this script in the `build and publish node` task, we can clean up all the logic in this script that looks at architecture versions to build.  the only logic that we need to keep is the package.json version check

Reply:
can we also remove the FLE platform matrix below or update it?  looks like it's outdated (we no longer support s390x)

Reply:
I've updated it to be accurate now.